R/theme-elements.R

```diff
@@ -276,6 +329,45 @@ rel <- function(x) {
   structure(x, class = "rel")
 }
 
+#' @export
+`$.ggplot2::element` <- function(x, i) {
+  # deprecate_soft0("4.0.0", I("`<ggplot2::element>$i`"), I("`<ggplot2::element>@i`"))
```

Comment:
We would wait for the 4.1.0 release with this, right?

Reply:
That's right, let me put the right version in these staged deprecations